Templates/local_openshift/hogarama-fluentd.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: hogarama-fluentd
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: fluentd-onbuild
+      template: hogarama-fluentd
+    name: fluentd-onbuild
+  spec:
+    lookupPolicy:
+      local: false
+    tags:
+    - annotations:
+        openshift.io/generated-by: OpenShiftWebConsole
+        openshift.io/imported-from: fluent/fluentd:edge-debian
+      from:
+        kind: DockerImage
+        name: fluent/fluentd:edge-debian
+      generation: 2
+      importPolicy: {}
+      name: edge-debian
+      referencePolicy:
+        type: Source
+  status:
```

Comment:
status section is also not needed in the export as it describes the current status of the imagestream/pod/build/etc. and is not needed for creation